src/main/java/frc/robot/Robot.java

```diff
@@ -279,28 +306,41 @@ public void teleopSetup() {
     elevator.start();
     shooter.start();
     ballSubsystem.start();
-  }
 
+
+  }
   /** This function is called periodically during operator control. */
   @Override
   public void teleopRoutine() {
-      double forward = this.driverPad.getValue(ControllerBindings.RIGHT_STICK, ControllerBindings.Y_AXIS);
-      double turn = this.driverPad.getValue(ControllerBindings.LEFT_STICK, ControllerBindings.X_AXIS);
+      double forward, turn;
+
+
+      forward = this.driverPad.getValue(ControllerBindings.LEFT_STICK, ControllerBindings.Y_AXIS);
+      turn = this.driverPad.getValue(ControllerBindings.RIGHT_STICK, ControllerBindings.X_AXIS);
```

Comment:
These sticks have been reversed compared to what the driver wanted at comp - forward on the right/Y and turn on the left/X, as seen in the deletion lines above in this diff chunk